.cirrus.yml

```diff
@@ -25,6 +25,7 @@ task:
   setup_script: ./dev/bots/cirrus_setup.sh
   matrix:
     - name: docs
+      skip: "!changesInclude('packages/flutter/**', 'packages/flutter_test/**', 'packages/flutter_drive/**', 'packages/flutter_localizations/**', 'packages/flutter_goldens/**', 'bin/internal/**') && $CIRRUS_BRANCH != 'master'"
```

Comment:
Looks like this is preventing docs from running on the stable branch, thereby keeping our API docs from updating on api.flutter.dev.

Reply:
i'm going to fix this on master